packages/kolibri-tools/lib/cli.js

```diff
@@ -320,7 +321,19 @@ program
         });
       }
     }
-    runWebpackBuild(mode, bundleData, mode === modes.DEV, options);
+
+    if (options.writeToDisk && mode === modes.DEV) {
+      cliLogging.warn(
+        'Enabling write-to-disk mode may fill up your developer machine with lots of different built files if frequent changes are made.'
+      );
+      runWebpackBuild(mode, bundleData, false, {
+        ...options,
+        cache: false,
```

Comment:
Is there a reason not to utilize the webpack cache here? It should help with build performance either way.

Also, the use of hot is limited to the devserver being active, which we are passing false for, so I think this `if` could do the warn, then we could update the runWebpackBuild to simply be:

```
runWebpackBuild(mode, bundleData, !options.writeToDisk && mode === modes.DEV, options);
```

and not have to invoke it separately in the if statement.

Reply:
yes, I thought cache will fill the dev's system

Reply:
So yes, if hot and cache are not being modified then i will run the webpack build with your suggestions. Thank You

Reply:
The cache is better managed than the writing built files to disk, so I wouldn't worry too much!

Reply:
Done!! Thank You